build.gradle

```diff
@@ -1,6 +1,9 @@
 plugins {
     // Apply the java plugin to add support for Java
     id 'java'
+
+    // For bringing in AWS dependencies
```

Comment:
Why do we need a Gradle plugin to bring in dependencies?  Maybe a bit more explanation in the comment?

Reply:
I followed the instructions here: https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-project-gradle.html. The first method (with the change to the `settings.gradle` file) doesn't work - `testCompile` can't find the dependencies it needs. The "legacy" approach, however, does work.

Reply:
I've added some text in a comment there that explains it.